test/channels_test.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:rtchat/models/channels.dart';
+
+// This is the very first time I write a test in flutter. take it easy on me plz! :D
+void main() {
+  test('Create Channels Test', () {
+    var channel1 = Channel("provider", "channelId", "displayName");
+    var channel2 = Channel("provider", "channelId", "another_display_name");
+    var channel3 = Channel("another_provider", "channelId", "displayName");
+    var channel4 = Channel("provider", "another_channelId", "displayName");
+    assert(channel1 == channel2);
```

Comment:
Instead of using `assert` in tests, use `expect(channel1, equals(channel2));` and so on below.